hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -445,7 +444,7 @@ public void testCreateTableCustomSortOrder() throws Exception {
 
   @Test
   public void testDatabaseAndNamespaceWithLocation() throws Exception {
-    Namespace namespace1 = Namespace.of("noLocation");
+    Namespace namespace1 = Namespace.of("nolocation");
```

Comment:
Is there a reason for this change?  Is there a change in behavior that we're working around?

Reply:
Yes, this is due to a change in behavior in Hive 4. Please see https://github.com/apache/iceberg/pull/12681 for details.